ms-inventory/routes/gets.go

```diff
@@ -11,10 +11,12 @@ import (
 )
 
 // InventoryGet allows for the retrieval of the entire inventory
-func InventoryGet(writer http.ResponseWriter, req *http.Request) {
+func (c *Controller) InventoryGet(writer http.ResponseWriter, req *http.Request) {
 	utilities.ProcessCORS(writer, req, func(writer http.ResponseWriter, req *http.Request) {
```

Comment:
`utilities.ProcessCORS` can be removed in a future PR as it is not built in to EdgeX, but has to be enabled